helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableReplica.java

```diff
@@ -37,6 +37,7 @@
 public class AssignableReplica implements Comparable<AssignableReplica> {
   private static final Logger LOG = LoggerFactory.getLogger(AssignableReplica.class);
 
+  private final String _replicaKey;
```

Comment:
_replicaId is more appropriate

Reply:
or simpler `String _id`

Reply:
Not really, the key is not unique, so it is not really an Id. I don't want to imply that this is a unique Id that one can use as a hash key.